parts/chapters/subsections/5.3/COMPS.fodt

```diff
@@ -4285,7 +4285,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:p text:style-name="P3161">1</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table2074.B2" office:value-type="string">
-       <text:p text:style-name="P5436">COMPS</text:p>
+       <text:p text:style-name="P5436"><text:a xlink:href="#__RefHeading___Toc27871_3671211675 Copy 1">COMPS</text:a></text:p>
```

Comment:
Here the parameter name COMPS has been interpreted as the keyword COMPS.

Reply:
I do not immediately see how to fix this with the current algorithm. However, we could handle special cases like this manually by inserting a marker that the script can recognize. For example, inserting a span tag with a non-existing `text:style-name`. The script can then check for that style name.  Like this: `<text:span text:style-name="NOT_KEYWORD">COMPS</text:span>`, where the style "NOT_KEYWORD" indicates to script that it should not link this word.